docs/prehashing.md

```diff
@@ -0,0 +1,167 @@
+# Telescope - Construction with Prehashing
+
+**Construction with Prehashing** optimizes the basic Telescope ALBA protocol by reducing the computational cost of checking sequences.
+Here, a sequence means an ordered set of elements or tuples that meet certain hash conditions in the protocol.
+In the basic approach, we would attempt to extend a given sequence "blindly" with all elements of $S_p$. 
```

Comment:
what do you mean by `blindly`?

Reply:
I think @rrtoledo means _without implementing the prehash oracle_, by `blindly`.

Reply:
By "blindy" I meant searching having no information and using all elements.